test/app_test.rb

```diff
@@ -1,15 +1,165 @@
 require "test_helper"
 
 class AppTest < Minitest::Test
+  def setup
+    @mock_app = Minesweeper::MockApp.new
+    @mock_game = Minesweeper::MockGame.new(5,5)
+    @mock_cli = Minesweeper::MockCli.new
+  end
 
   def test_that_it_has_an_app_class
-    app = Minesweeper_2pl::App.new
-    refute_nil app
+    refute_nil @mock_app
```

Comment:
Can you describe in words what this test is testing? If you deleted this test, would you be less confident that the production code is working correctly? Why?